search-app/src/cmr/search/services/query_service.clj

```diff
@@ -200,38 +200,52 @@
   [context concept-type params]
   (let [tag-data (make-concepts-tag-data params)
         [query-creation-time query] (u/time-execution
-                                      (make-concepts-query
-                                        context concept-type params tag-data))
+                                     (make-concepts-query
+                                      context concept-type params tag-data))
         [find-concepts-time results] (u/time-execution
-                                       (common-search/find-concepts
-                                         context concept-type query))
+                                      (common-search/find-concepts
+                                       context concept-type query))
         total-took (+ query-creation-time find-concepts-time)
-        short-scroll-id (str (hash (:scroll-id results)))
+        scroll-id (:scroll-id results)
+        search-after (:search-after context)
         log-message (format "Found %d %ss in %d ms from client %s in format %s with params %s"
-                      (:hits results) (name concept-type) total-took (:client-id context)
-                      (rfh/printable-result-format (:result-format query)) (pr-str params))]
-    (info (if (string/blank? short-scroll-id)
-              (format "%s." log-message)
-              (format "%s, scroll-id: %s." log-message short-scroll-id)))
+                            (:hits results) (name concept-type) total-took (:client-id context)
+                            (rfh/printable-result-format (:result-format query)) (pr-str params))]
+    (info (if scroll-id
+            (format "%s, scroll-id: %s." log-message (str (hash scroll-id)))
+            (if search-after
+              (format "%s, search-after: %s, new search-after: %s."
+                      log-message
+                      (json/encode search-after)
+                      (:search-after results))
+              (format "%s." log-message))))
```

Comment:
Not sure how you feel about `cond`, but something like this reads more clearly to me than a nested `if`

```suggestion
    (info (cond
            scroll-id (format "%s, scroll-id: %s." log-message (str (hash scroll-id)))
            search-after (format "%s, search-after: %s, new search-after: %s."
                                 log-message
                                 (json/encode search-after)
                                 (:search-after results))
            :else (format "%s." log-message))))
```